src/lib.rs

```diff
@@ -20,7 +20,7 @@ the rust programming language.
 #![allow(clippy::type_complexity)] // Complains about closures that are fairly simple.
 #![doc(html_root_url = "http://docs.rs/parry/0.1.1")]
 #![cfg_attr(not(feature = "std"), no_std)]
-#![cfg_attr(not(feature = "rkyv"), deny(unused_qualifications))] // TODO: deny that everytime
+#![deny(unused_qualifications)] // TODO: deny that everytime
```

Comment:
I believe that's fixed by https://github.com/dimforge/parry/pull/273/files#diff-539eccbdd3d52ed1a28671920995e8af3f052e319dabb925b39308a924bc5231R126